### ISSUE TITLE
Tabs not coded according to best practice

### ISSUE THREAD

**Issue by plannero** [umbraco/Umbraco-CMS#14742]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

10.4.0

### Bug summary

My screen reader (VoiceOver) does not present the tab system correctly (the one that contain the main editing areas).

### Specifics

The tab system has been marked up with `role=”tablist”` and `role=”tab”`. This is a good start, but in order to become truly accessible to users with screen readers a bit more markup is needed. For example, the tab panels should have `role=”tabpanel”` and the tablist must have a name (either visible or via `aria-label`). 
**Please** see (and implement) https://www.w3.org/WAI/ARIA/apg/patterns/tabs/ 

<img width="770" alt="image" src="https://github.com/umbraco/Umbraco-CMS/assets/2157085/38da5f7e-23c2-4d19-9361-f2ca79f676f9">

### Steps to reproduce

Edit a page using a screen reader and try to understand the structure of the editor. This is very hard, since you are not able to set focus on the tablist (since it has no name).

### Expected result / actual result

A screen reader user should be able to navigate to major components of the web page, such as the tab system, and hear them presented in a meaningful way.

**Comment by andr317c** [comment]
Hey! 😃 

Thanks for raising this issue! This is something we would like help with, I'll set this issue as up for grabs 😄

**Comment by nul800sebastiaan** [comment]
Fixed in https://github.com/umbraco/Umbraco-CMS/commit/3553e7546afbead882a90dfd28f11f63586c9a23
